webdriver/tests/support/fixtures_bidi.py

```diff
@@ -65,19 +65,17 @@ async def execute_as_async(sync_func, **kwargs):
 async def subscribe_events(bidi_session):
     subscriptions = []
 
-    async def subscribe_events(events, contexts=None):
-        result = await bidi_session.session.subscribe(events=events, contexts=contexts)
-        subscriptions.append((events, contexts))
+    async def subscribe_events(events, contexts=None, user_contexts=None):
+        result = await bidi_session.session.subscribe(events=events, contexts=contexts, user_contexts=user_contexts)
+        subscriptions.append(result["subscription"])
         return result
 
     yield subscribe_events
 
-    for events, contexts in reversed(subscriptions):
-        try:
-            await bidi_session.session.unsubscribe(events=events,
-                                                   contexts=contexts)
-        except (InvalidArgumentException, NoSuchFrameException):
-            pass
+    try:
+        await bidi_session.session.unsubscribe(subscriptions=subscriptions)
+    except (InvalidArgumentException, NoSuchFrameException):
```

Comment:
```suggestion
    except NoSuchFrameException:
```

I don't think we can get `NoSuchFrameException` anymore with unsubscribe, we remove it